pyEpiabm/pyEpiabm/sweep/intervention_sweep.py

```diff
@@ -0,0 +1,50 @@
+#
+# Sweeps for taking care of the interventions
+#
+
+from pyEpiabm.core import Parameters
+from pyEpiabm.intervention import Vaccination
+
+from .abstract_sweep import AbstractSweep
+
+
+class InterventionSweep(AbstractSweep):
+    """Class to sweep through all possible interventions.
+    Check if intervention should take place based on time (and/or threshold).
+    Possible interventions:
+    vaccination: Implement mass vaccination
+    """
+
+    def __init__(self):
+        """Call in variables from the parameters file and set flags
```

Comment:
I can't see any flags being set here? Also strictly speaking we are reading in, not calling in